src/core_plugins/getting_started/public/components/getting_started/getting_started.js

```diff
@@ -0,0 +1,82 @@
+import { uiModules } from 'ui/modules';
+import uiChrome from 'ui/chrome';
+import 'ui/storage';
+import 'ui/filters/trust_as_html';
+import { getTopMessage, getManageAndMonitorMessages } from 'ui/getting_started/registry';
+import { hasOptedOutOfGettingStarted, optOutOfGettingStarted } from 'ui/getting_started/opt_out_service';
+import { documentationLinks } from 'ui/documentation_links';
+import angular from 'angular';
+
+import kibanaLogo from 'ui/images/logo-kibana-small.svg';
+import beatsLogo from 'ui/images/logo-beats-small.svg';
+import logstashLogo from 'ui/images/logo-logstash-small.svg';
+import dashboardIcon from 'ui/images/icon-dashboard.svg';
+import shieldIcon from 'ui/images/icon-shield.svg';
+
+import template from './getting_started.html';
+import './getting_started.less';
+
+function makeAngularParseableExpression(text) {
+  return `<span>${text}</span>`;
+}
+
+const app = uiModules.get('kibana');
+
+app.directive('gettingStarted', function ($injector) {
+
+  const $compile = $injector.get('$compile');
+
+  return {
+    restrict: 'E',
+    template: template,
+    scope: {
+    },
+    bindToController: true,
+    controllerAs: 'gettingStarted',
+    controller: class GettingStartedController {
+      constructor($scope) {
+        if (this.hasOptedOut()) {
+          uiChrome.setVisible(true);
+        } else {
+          uiChrome.setVisible(false);
+        }
+
+        // Compile topMessageHtml and inject it into the DOM
+        const topMessage = getTopMessage();
+        if (topMessage) {
+          const topMessageContainer = document.getElementById('gettingStartedTopMessageContainer');
```

Comment:
Yes, this is deliberate because `topMessage` can contain an angular expression, e.g. a directive. Injecting that the normal way (as just a variable on the scope) doesn't cause Angular to reevaulate the template and render any new Angular expressions.

AFAIK, this is the only way to achieve what we want but I might be missing something and will gladly consider a more intuitive approach.